data/hails.txt

```diff
@@ -317,8 +321,11 @@ phrase "friendly civilian"
 		"my crew is"
 		"some of my friends are"
 		"all the oldsters are"
+		"they said they're"
+		"
```

Comment:
Is this " intended? Its likely the origin of your "closing quote missing" trace.

Reply:
I believe it may have been, but I actually did fix it already but didn't commit the change yet, as I'm having a problem testing it. for some reason I keep getting the following, which prevents me from launching the game: Unable to query monitor resolution! (SDL message: "Video subsystem has not been initialized")

any idea what caused this? I didn't change anything since I made the last commit and I've been getting this error since.

Reply:
That sounds worthy of its own post or some internet research. 
From a short tour of the game's `main.cpp` file I can tell you that is likely an error from SDL_Init() as it isn't one the game gives you.